compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -2362,6 +2376,26 @@ fn add_upstream_rust_crates<'a, B: ArchiveBuilder<'a>>(
                             (lib.name, lib.kind, lib.verbatim)
                         };
 
+                        if sess.opts.unstable_opts.split_bundled_libs {
+                            if let NativeLibKind::Static {
```

Comment:
Could you merge this `if let` and `if let` below into a single exhaustive `match` like in https://github.com/rust-lang/rust/pull/99467?